scripts/qa_crowbarsetup.sh

```diff
@@ -3667,7 +3667,23 @@ function oncontroller_upload_defcore
     yes | refstack-client/refstack-client upload-subunit --keystone-endpoint $OS_AUTH_URL --url https://10.86.0.98 --insecure tempest.subunit_1.log
     popd
 }
+function onadmin_upload_tempest_results
+{
+    local scenario_name=$1
+    local hw_number=$2
+
+    #generate subunit results
+    ssh controller1 "pushd /var/lib/openstack-tempest-test;stestr last --subunit > $scenario_name"
+
+    #copy to server in provo
+    scp controller1:/var/lib/openstack-tempest-test/$scenario_name .
+    scp $scenario_name ubuntu@10.84.144.252:/var/www/html/tempest-crowbar/
 
```

Comment:
Perhaps use rsync here.

Reply:
Also, since you run it on admin, you need to generate and copy crowbar key to the 10.84.144.252. is it done?

Reply:
Plan update all admin vm images with copied keys but its outside this PR.